eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -126,7 +123,7 @@ stages:
                       - task: MSBuild@1
                         displayName: 'Upload Symbols for ${{artifact.name}}'
                         inputs:
-                          solution: '$(AzureSDKBuildToolsPath)/tools/symboltool/SymbolUploader.proj'
+                          solution: 'tools/symboltool/SymbolUploader.proj'
```

Comment:
This assumes the working directory is under the azure-sdk-build-tools clone, I'm not sure if that is the case.

Reply:
Updated